generated-ascii-files/prebuilt-rules-changelog.asciidoc

```diff
@@ -0,0 +1,1263 @@
+[[prebuilt-rules-changelog]]
```

Comment:
remove these `generated-ascii-files/*`

Reply:
Yes I think these snuck their way in from the previous build for Kibana. Removed.